src/main/java/uk/gov/companieshouse/pscfiling/api/model/entity/PscIndividualFiling.java

```diff
@@ -11,110 +13,67 @@
 import java.util.function.Consumer;
 import org.springframework.data.annotation.Id;
 import org.springframework.data.mongodb.core.mapping.Document;
-import org.springframework.data.mongodb.core.mapping.Unwrapped;
 
 @Document(collection = "psc_submissions")
 @JsonInclude(JsonInclude.Include.NON_NULL)
-public class PscIndividualFiling implements PscCommunal {
+public class PscIndividualFiling extends PscCommon implements PscCommunal, Touchable {
+    @Override
+    public boolean equals(final Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+        if (!super.equals(o)) {
+            return false;
+        }
+        final PscIndividualFiling that = (PscIndividualFiling) o;
+        return Objects.equals(getId(), that.getId())
+                && Objects.equals(getCountryOfResidence(), that.getCountryOfResidence())
+                && Objects.equals(getDateOfBirth(), that.getDateOfBirth())
+                && Objects.equals(getNameElements(), that.getNameElements())
+                && Objects.equals(getNationality(), that.getNationality())
+                && Objects.equals(getResidentialAddress(), that.getResidentialAddress())
+                && Objects.equals(getResidentialAddressSameAsCorrespondenceAddress(),
+                that.getResidentialAddressSameAsCorrespondenceAddress())
+                && Objects.equals(getStatementActionDate(), that.getStatementActionDate())
+                && Objects.equals(getStatementType(), that.getStatementType());
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(super.hashCode(), getId(), getCountryOfResidence(), getDateOfBirth(),
+                getNameElements(), getNationality(), getResidentialAddress(),
+                getResidentialAddressSameAsCorrespondenceAddress(), getStatementActionDate(),
+                getStatementType());
+    }
 
     @Id
+    @JsonProperty(access = JsonProperty.Access.READ_ONLY)
     private String id;
-    @Unwrapped.Empty
-    private PscCommon pscCommon;
     private String countryOfResidence;
+    @JsonMerge
     private Date3Tuple dateOfBirth;
+    @JsonMerge
     private NameElements nameElements;
     private String nationality;
+    @JsonMerge
     private Address residentialAddress;
     private Boolean residentialAddressSameAsCorrespondenceAddress;
     private LocalDate statementActionDate;
     private String statementType;
 
     public PscIndividualFiling() {
         // required by Spring JPA
-        pscCommon = PscCommon.builder()
+        PscCommon.builder()
                 .build();
     }
```

Comment:
Please remove this redundant builder call, but keep the public constructor and the comment